specifications/api/swagger.yaml

```diff
@@ -35,12 +35,27 @@ numberOfSlots: &numberOfSlots
   properties:
     quantity:
       type: integer
-      minimum: 0
+      minimum: 1337
     unit:
       type: string
       enum:
         - slot
-      example: 1337
+      example: "slot"
+
+numberOfBlocks: &numberOfBlocks
+  type: object
+  required:
+    - quantity
+    - unit
+  properties:
+    quantity:
+      type: integer
+      minimum: 1337
+    unit:
+      type: string
+      enum:
+        - block
+      example: "block"
```

Comment:
additionally to represent "depth" concept we have added a seperate depth type to swagger. It might not be worth to increase complexity of swagger.json just for this single type?

We have discussed a bit in previous PRs and/or slack that concept of depth (ie, number of confirmation) should probably be defined in terms of blocks (non-empty blockchain slots - where work has actually happend to produce the block).

If we think additional complexity is not worth it I will remove it

Reply:
I have changed `last_blocks` unit type from "slots" to "blocks" in https://github.com/input-output-hk/cardano-wallet/pull/139/commits/e472f21b55f9672359bbc14a8518f68b7c3a54e4 as it makes sense from the documentation.

Person implementing this feature will eventually change if this unit type is not suitable